tuplex/core/include/ee/local/LocalBackend.h

```diff
@@ -184,6 +184,27 @@ namespace tuplex {
      * @return
      */
     extern URI outputURI(const UDF& udf, const URI& baseURI, int64_t partNo, FileFormat fmt);
+
+    /*!
+     * Trim list of partitions so that it includes up to the first n rows and the last m rows
```

Comment:
can you add a sentence what happens in the case that n + m > #rows stored?

Reply:
are there special values for topLimit/bottomLimit (i.e. numeric_limits::max() e.g. ?)